### DISCUSSION TITLE
how do i declare `I18nT` types in `components.d.ts` file in order to get ts suggest

### DISCUSSION THREAD

**Discussion by jhxxs** [intlify/vue-i18n#835] [Q&A]
right now i do it like this:
```ts
declare module "vue" {
  export interface GlobalComponents {
    RouterLink: typeof import("vue-router")["RouterLink"]
    RouterView: typeof import("vue-router")["RouterView"]
    I18nT: typeof import("vue-i18n")["Translation"]
  }
}

export {}

```
`RouterView` and `RouterLink` can get autosuggest in sfc, unfortunately when i use Component Interpolation in sfc, i can't get any suggestion.
thanks for help😢

**Comment by santiagoarizti** [comment]
Hi, I just added a new discussion [here](https://github.com/intlify/vue-i18n-next/discussions/872)

I proposed to allow import the `i18n-t` component `from `vue-i18n`, what do you think? this should allow typescript or any IDE to easily autocomplete for the component.
